src/App.tsx

```diff
@@ -1,7 +1,33 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    keyPressed: undefined,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ keyPressed: event.key });
+    });
+  }
+
```

Comment:
due to the task, you have to also use removeEventListener to remove a global handler in componentWillUnmount